#std_msgs/Header header
uint8 type
float32 confidence
float32[] real
float32[] projection
float32[] orientation
